src/pages/index.js

```diff
@@ -1,11 +1,13 @@
 import Card from "../components/Card.js";
 import Section from "../components/Section.js";
 import UserInfo from "../components/UserInfo.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import FormValidator from "../components/FormValidator.js";
```

Comment:
Please, delete `this.disableButton();` from line 80 from `FormValidator` because you need to disable buttons only in block `then` otherwise you'll not be able to click it again in case of a server error

Reply:
can you elaborate on how this and how I might change it? I put in an incorrect endpoint but the modal still functioned as it should. It closed and reset even with the incorrect endpoint, and I even had someone look at it in coworking hours and they did not know how to fix the problem you are referring to. Thank you!

Reply:
> can you elaborate on this issue and how I might change it? I put in an incorrect endpoint but the modal still functioned

I mean this line https://snipboard.io/RMiyo0.jpg . In the 9th sprint you need to disable the button only in `then` (not right after submitting)

Reply:
Okay, I think I got it. Please let me know if there is anything else that needs to be corrected. Thank you for your time!